mysql-test/suite/rpl/r/rpl_mixed_binlog_max_cache_size.result

```diff
@@ -1,6 +1,8 @@
 include/master-slave.inc
 [connection master]
 call mtr.add_suppression("Unsafe statement written to the binary log using statement format since BINLOG_FORMAT = STATEMENT");
+call mtr.add_suppression("Write to binary log failed: Multi-row statements required more than .max_binlog_stmt_cache_size.* ");
```

Comment:
Ah, when the patch was originally written, `mysql-test/extra/rpl_tests` existed and the extension to `rpl_binlog_max_cache_size.test` was just the addition of two `call mtr.add_suppression` lines. Since then, it was moved into `suite/rpl/include`; yet this patch didn't move the file and re-created the `extra/rpl_tests` directory. When adding the lines into the correct place in `suite/rpl/include/rpl_binlog_max_cache_size.test`; they are required again. I updated this in my newest commit